lib/eunit/src/eunit_lib.erl

```diff
@@ -49,20 +44,20 @@
 	 format_exception/1, format_exception/2, format_error/1, format_error/2,
          format_stacktrace/1, is_not_test/1]).
 
+-export_type([exception/0, exceptionClass/0, stackTrace/0, moduleName/0,
+              functionName/0, argList/0, fileName/0]).
+
 -define(DEFAULT_DEPTH, 20).
 
 %% Type definitions for describing exceptions
-%%
-%% @type exception() = {exceptionClass(), Reason::term(), stackTrace()}
-%%
-%% @type exceptionClass() = error | exit | throw
-%%
-%% @type stackTrace() = [{moduleName(), functionName(), arity() | argList()}]
-%%
-%% @type moduleName() = atom()
-%% @type functionName() = atom()
-%% @type argList() = [term()]
-%% @type fileName() = string()
+
+-type exception() :: {exceptionClass(), Reason::term(), stackTrace()}.
+-type exceptionClass() :: error | exit | throw.
+-type stackTrace() :: [{moduleName(), functionName(), arity() | argList()}].
+-type moduleName() :: atom().
+-type functionName() :: atom().
+-type argList() :: [term()].
+-type fileName() :: string().
```

Comment:
We could align those to snake_case as well, and use erlang.erl's -type module().